projects/project-2/readings/Project 2 - SPARQL library

```diff
@@ -35,6 +39,10 @@ It uses FILTER NOT EXISTS to ensure that only classes without any defined instan
 
 2. Identifying Disjoint Classes with Common Instances
 
+Title: Disjoint Class Violation
+Constraint Description: Detects instances that belong to classes declared as disjoint.
+Severity: Error
+
 PREFIX owl: <http://www.w3.org/2002/07/owl#>
 PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
 SELECT ?class1 ?class2 ?instance
```

Comment:
you should consider whether you want to include blank nodes or not. This probably holds for most of the queries here

Reply:
Also, you should consider whether you also want to add error messages

Reply:
What level is this query?

Reply:
The levels are indicated in the start of the document.